translations/es/10-glosario.md

```diff
@@ -0,0 +1,72 @@
+
+## Incorporación
+
+La documentación de incorporación ayuda a los nuevos miembros o colaboradores del equipo a familiarizarse rápidamente con la estructura, los objetivos y los procesos de un proyecto.
+
+## Fusionar
+
+La fusión es el proceso de combinar cambios de una rama en otra. Cuando se acepta y fusiona una solicitud de extracción, los cambios realizados en la solicitud de extracción pasan a formar parte del código base principal.
+
+## Guía de estilo
+
+Una guía de estilo es un conjunto de reglas y convenciones que definen el formato preferido, el estilo de escritura y los elementos visuales utilizados en la documentación y otro contenido. Esto ayuda a mantener la coherencia y la claridad en todos los documentos, haciéndolos más fáciles de leer y comprender.
+
+
+## Mantenedor
+
+Un mantenedor es una persona o un grupo de personas responsables de mantener un proyecto Open Source específico. Los mantenedores suelen ser responsables de revisar y aceptar o rechazar las contribuciones de otros contribuyentes. También tienen la autoridad para tomar decisiones finales sobre la dirección y el alcance del proyecto.
+
+## Markdown
+
+Markdown es un lenguaje de marcado ligero que se utiliza habitualmente para crear documentos de texto formateados. Es ampliamente utilizado para crear documentación y archivos README en el desarrollo de software debido a su simplicidad y legibilidad.
+
+## Miembro principal
+
+Un miembro principal es un colaborador al que se le han otorgado privilegios o responsabilidades adicionales dentro de un proyecto Open Source. Los miembros principales suelen ser colaboradores confiables que han demostrado un profundo conocimiento del proyecto y han realizado contribuciones significativas a su desarrollo.
+
+## Notas de la versión
+
+Las notas de la versión son documentos que detallan cambios, mejoras, correcciones de errores y nuevas funciones en cada versión del software. Informan a los usuarios y a las partes interesadas sobre qué esperar de una nueva versión del software.
+
+## Problema
+
+Un problema es un problema o error que debe abordarse en el código. Cualquiera puede crear problemas y, a menudo, se utilizan para realizar un seguimiento de errores, solicitudes de funciones y otras tareas que deben realizarse.
+
+## Proyectos OSS
+
+Proyectos OSS significa proyectos de "software Open Source". Se trata de proyectos de software en los que el código fuente se pone a disposición del público, lo que permite a cualquiera ver, utilizar y modificar el software.
```

Comment:
[suggestion] "I would change “software Open Source” back to “Open Source Software” since we're still providing the English abbreviation "OSS”.
@Mike-Salcedo, this is what I mean by implementing feedback 
```suggestion
Proyectos OSS significa proyectos de " Open Source Software". Se trata de proyectos de software en los que el código fuente se pone a disposición del público, lo que permite a cualquiera ver, utilizar y modificar el software.
```